finding/finding_test.go

```diff
@@ -198,3 +199,88 @@ func Test_FromBytes(t *testing.T) {
 		})
 	}
 }
+
+func TestOutcome_UnmarshalYAML(t *testing.T) {
+	type args struct {
+		n *yaml.Node
+	}
+	tests := []struct { //nolint:govet
+		name    string
+		o       Outcome
```

Comment:
wouldn't `o Outcome` be `want Outcome` in this case?

Reply:
👍